paddle/fluid/operators/elementwise/elementwise_op_function.h

```diff
@@ -50,22 +50,56 @@ namespace operators {
  */
 inline void get_mid_dims(const framework::DDim &x_dims,
                          const framework::DDim &y_dims, const int axis,
-                         int *pre, int *n, int *post) {
+                         int *pre, int *n, int *post, int *mid_flag = NULL) {
```

Comment:
Add a comment of mid_flag.
And add an example of it in the above comment.

Reply:
Done.